client/devdocs/design/index.jsx

```diff
@@ -68,6 +68,8 @@ export default React.createClass( {
 	render() {
 		return (
 			<div className="design-assets" role="main">
+				<h1>Design Reference</h1>
```

Comment:
We could drop it; I like it because it matches other DevDocs pages visually. And, it was requested by Nikolay.